speed/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+	"extends": "../.eslintrc.json",
+	"globals": {
+		"console":  true,
+		"require": true,
+		"setTimeout": true,
+		"location": true,
```

Comment:
Could you remove the `location` & `setTimeout` entries from here & change them in `speed/*` code to `window.location` & `window.setTimeout`? That'd improve compatibility with non-browser environments.

Let me know if that's too much work.

Reply:
No problem. Done